.circleci/config.yml

```diff
@@ -145,7 +153,20 @@ workflows:
 
   client-python-release:
     jobs:
+      - release-github:
+          filters:
+            branches:
+              only: client-python-release-branch
+      - release-approval:
```

Comment:
let's rename this to `release-deployment-approval` so we don't get confused with the other `release-approval` which comes before the whole release workflow.

Reply:
@haikalpribadi addressed in https://github.com/graknlabs/client-python/pull/36/commits/224287ccbff9b509cb7a8ae4ac46d6eb0849dbd6